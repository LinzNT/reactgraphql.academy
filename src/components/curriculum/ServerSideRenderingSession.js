import React from 'react'
import Ul, { Li } from '../layout/Ul'
import Session from './Session'

const ServerSideRenderingSession = ({ title }) => (
  <Session title={title}>
    <Ul>
      <Li>Nodejs + React + React Router</Li>
      <Li>Universal Redux</Li>
      <Li>GraphQL SSR</Li>
      <Li>Styled-componets SSR</Li>
    </Ul>
  </Session>
)

export default ServerSideRenderingSession
