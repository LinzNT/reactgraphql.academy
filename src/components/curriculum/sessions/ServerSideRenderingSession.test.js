import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import ServerSideRenderingSession from './ServerSideRenderingSession'

describe('<ServerSideRenderingSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<ServerSideRenderingSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
