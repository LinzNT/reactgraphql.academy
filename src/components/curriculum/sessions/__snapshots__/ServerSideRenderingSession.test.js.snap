// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServerSideRenderingSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Webpack
    </styled.li>
    <styled.li>
      Nodejs + React + React Router
    </styled.li>
    <styled.li>
      Universal Redux
    </styled.li>
    <styled.li>
      GraphQL SSR
    </styled.li>
    <styled.li>
      Styled-componets SSR
    </styled.li>
  </styled.ul>
</Session>
`;
